aws-cpp-sdk-core/source/http/windows/WinINetSyncHttpClient.cpp

```diff
@@ -115,6 +106,15 @@ void* WinINetSyncHttpClient::OpenRequest(const Aws::Http::HttpRequest& request,
         ss.str().c_str(), nullptr, nullptr, accept, requestFlags, 0);
     AWS_LOGSTREAM_DEBUG(GetLogTag(), "HttpOpenRequestA returned handle " << hHttpRequest);
 
+    if (!m_verifySSL)
+    {
+        AWS_LOG_WARN(GetLogTag(), "Turning ssl unknown ca verification off.");
+        DWORD flags = SECURITY_FLAG_IGNORE_UNKNOWN_CA | INTERNET_FLAG_IGNORE_CERT_CN_INVALID;
+
+        if (!InternetSetOptionA(hHttpRequest, INTERNET_OPTION_SECURITY_FLAGS, &flags, sizeof(flags)))
```

Comment:
I can't imagine this is correct. The connection has already been made at this point. Am I missing something in the documentation?